cg_hermes/config/balsamic.py

```diff
@@ -402,6 +406,26 @@
         "is_mandatory": False,
         "used_by": ["deliver", "scout"],
     },
+    frozenset(RAW_TAGS["cov.bed.gz"]): {
+        "tags": ["gens", "coverage", "bed"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(RAW_TAGS["cov.bed.gz.tbi"]): {
+        "tags": ["gens", "coverage", "bed-index"],
```

Comment:
I'm confused about some of these tags, I see that this "bed-index" is in the MIP config as well, but I can't see the index in housekeeper attached to these index files, just:
`coverage, bed, scout, gens, [LIMS-ID]`

Reply:
I'm not sure why we started to add `bed-index` for MIP but it's not a key in the model, so it doesn't do anything. That's why we have the same tags for both the bed and the index files in rare diseases.

Regarding the LIMS ID, `get_balsamic_files` add the "id" value from the deliverables files as tag (independently if its the case or sample ID)